warehouse/query-core/src/main/java/datawave/query/postprocessing/tf/TermOffsetPopulator.java

```diff
@@ -153,34 +154,33 @@ public Map<String,Object> getContextMap(Key docKey, Set<Key> keys, Set<String> f
         if (evaluationFilter != null) {
             evaluationFilter.startNewDocument(docKey);
         }
-        
+
+        Key key;
+        Content attr;
+        TermFrequencyKey parser = new TermFrequencyKey();
+        TermWeight.Info twInfo;
+        TermFrequencyList.Zone twZone;
+        TermWeightPosition.Builder position = new TermWeightPosition.Builder();
```

Comment:
what is the benefit of defining these variables here instead of within the loop?  Aside from parser and position, we are creating new instances of these with each loop anyway...

Reply:
Great point, moved variables back inside the loop except for the parser and position builder